training/distributed_training/pytorch/model_parallel/gpt-j/01_train_gptj_smp_notebook.ipynb

```diff
@@ -2,6 +2,7 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    instance_type = "ml.p4d.24xlarge"
</pre><p>Like in the other notebook, explain how customers can get access to this instance type.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3360/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
added the steps to request a limit increase from AWS Support